actions/codeowners-review-analysis/src/github.ts

```diff
@@ -100,6 +100,59 @@ export async function getCodeownersFile(
   return undefined;
 }
 
+export async function getTeamToMembersMapping(
+  octokit: OctokitType,
+  org: string,
+  teams: string[],
+) {
+  core.info(`Fetching members for ${teams.length} teams...`);
+
+  const teamToMembers: Map<string, string[]> = new Map();
+  for (const team of teams) {
+    const [_, slug] = team.split("/");
+    const members = await getTeamMembers(octokit, org, slug);
+    if (members.length === 0) {
+      core.warning(`No members found for team: ${org}/${slug}`);
+      continue;
+    }
+
+    core.info(`Found ${members.length} members for team: ${org}/${slug}`);
+    core.debug(`Members: ${JSON.stringify(members, null, 2)}`);
```

Comment:
Is team membership public? If not, should we avoid logging this?

Reply:
Removed logging of any team membership unless `CL_LOCAL_DEBUG` env var is `true`.

8c2032aa191de1e14b4b93a9a6db4bf84a24f4ce